### ISSUE TITLE
Problem if username in local language (non english)

### ISSUE THREAD

**Issue by testzcrypto** [bitshares/bitshares1-core#791]
Tested version 0.4.15-a Windows 8 (user reports same for Windows 7).

If user name in local language (for example Тест), BitSharesX start with error "An error occurred while trying to start", if you run BitSharesX as administrator it's start but creates 2 additional folders in C:\Users:
"C:\Users\Р СћР ВµРЎРѓРЎвЂљ\AppData\Roaming\BitShares X\logs" 
"C:\Users\РўРµСЃС‚\AppData\Roaming\BitShares X"

And no "BitShares X" folder at C:\Users\Тест\AppData\Roaming

Looks like problem located in handling Unicode paths in different BitShares modules.

**Comment by nathanhourt** [comment]
Is this still happening? Have you tried it with a fresh data dir?

**Comment by testzcrypto** [comment]
Yes, it's still happening, it's not a problem with fresh data dir, the problem is local characters in the path, if your Windows user name uses local characters (like Тест) you will see the problem, probably same will happened if you create folder C:\Тест and specify it as --data-dir.

**Comment by nathanhourt** [comment]
Right, I understand, but I've had an issue before where logs weren't being generated correctly because my config.json was generated by an outdated client.

**Comment by testzcrypto** [comment]
Just check and I can confirm that this problem happening with v0.4.27.2, it's show error message, no data dir, no log files (%APPDATA% = C:\Users\Тест\AppData\Roaming) if I specify --data-dir - works good.

**Comment by nathanhourt** [comment]
If you specify a data dir with non-ASCII characters, it works?

**Comment by vikramrajkumar** [comment]
@testzcrypto Is this problem still present in 0.6.x version?

**Comment by testzcrypto** [comment]
Yes, problem still present in 0.6.2.

**Comment by vikramrajkumar** [comment]
Possibly related: https://github.com/BitShares/qt_wallet/issues/94

**Comment by testzcrypto** [comment]
Reproducible in BitShares 0.9.1, Windows username Тест, %APPDATA% = C:\Users\\**Тест**\\AppData\Roaming

**Comment by testzcrypto** [comment]
Probably same problem with French Windows user name:
https://bitsharestalk.org/index.php/topic,16235.msg
